go.mod

```diff
@@ -1,8 +1,13 @@
 module github.com/networkservicemesh/networkservicemesh
 
 // Use ./scripts/switch_k8s_version.sh to change k8s version
+replace gonum.org/v1/gonum => github.com/gonum/gonum v0.0.0-20190331200053-3d26580ed485
```

Comment:
@nickolaev this two guys are not accessible from Russia anymore, not sure why. Checked from few places.